### PR TITLE
Establish Order Conventions for ember Routes

### PR DESCRIPTION
**Pull request by kculafic**
These conventions will be re-asserted by ESLint (https://github.com/ember-cli/eslint-plugin-ember).

For reference, ESLint's default `config` is as follows: 
```
ember/order-in-routes: [2, {
  order: [
    'service',
    'inherited-property',
    'property',
    'single-line-function',
    'multi-line-function',
    'beforeModel',
    'model',
    'afterModel',
    'serialize',
    'redirect',
    'activate',
    'setupController',
    'renderTemplate',
    'resetController',
    'deactivate',
    'actions',
    ['method', 'empty-method'],
  ]
}]
``` 

This Pull Request is to confirm our team's preferences, which is a prerequisite to updating our ember route style conventions (https://www.pivotaltracker.com/story/show/150684734) .

### CODE REVIEW

**Review by amiel — APPROVED** [review]
This looks good to me. I am happy with the default ordering defined by the linter.

Thanks for starting the conversation here @kculafic. As before, please don't merge just because I accepted. I think it'd be best to wait for input from all of the people who work with Ember regularly.

**Review by nathancarnes — APPROVED** [review]
Looks good to me!

File: ember.md
Comment (andreafrost) [inline ember.md]: This section should be numbered so it matches the numbering below.

Please use the following instead:

```
1. Services
1. Default route's properties
```
etc...

**Review by andreafrost — CHANGES_REQUESTED** [review]

File: ember.md
Comment (kculafic) [reply ember.md]: @andreafrost Nice catch! I agree numbers are better 👍

**Review by andreafrost — APPROVED** [review]
Thanks for making that change @kculafic :)

**Review by maprules1000 — COMMENTED** [review]
👍

**Pull request merged**
